src/components/LoanCards/LoanMatchingText.vue

```diff
@@ -19,7 +19,7 @@
 				:style="{ padding: '4px 0 2px 0', height: '1.2rem'}"
 			>
 		</span>
-		<h4 class="tw-inline-block">
+		<h4 class="tw-inline-block" style="color: #A24536;">
```

Comment:
As it stands this will show up on the homepage, Managed Lending pages and any other locations that use KivaClassicBasicLoanCard. I don't think this change was meant to go site wide was it...

Reply:
I checked, and they were OK with the propagation. But I'll ask again just in case.

Reply:
Team decided to revert the color change for now.